metadata/nz_building_outlines_lifecycle.xml

```diff
@@ -0,0 +1,331 @@
+                <gco:CharacterString>New Zealand</gco:CharacterString>
+              </gmd:country>
+              <gmd:electronicMailAddress>
+                <gco:CharacterString>info@linz.govt.nz</gco:CharacterString>
+              </gmd:electronicMailAddress>
+            </gmd:CI_Address>
+          </gmd:address>
+        </gmd:CI_Contact>
+      </gmd:contactInfo>
+      <gmd:role>
+        <gmd:CI_RoleCode codeList="http://asdd.ga.gov.au/asdd/profileinfo/gmxCodelists.xml#CI_RoleCode" codeListValue="resourceProvider">resourceProvider</gmd:CI_RoleCode>
+      </gmd:role>
+    </gmd:CI_ResponsibleParty>
+  </gmd:contact>
+  <gmd:dateStamp>
+    <gco:Date>2019-04-17</gco:Date>
+  </gmd:dateStamp>
+  <gmd:metadataStandardName>
+    <gco:CharacterString>ANZLIC Metadata Profile: An Australian/New Zealand Profile of AS/NZS ISO 19115:2005, Geographic information - Metadata</gco:CharacterString>
+  </gmd:metadataStandardName>
+  <gmd:metadataStandardVersion>
+    <gco:CharacterString>1.1</gco:CharacterString>
+  </gmd:metadataStandardVersion>
+  <gmd:referenceSystemInfo>
+    <gmd:MD_ReferenceSystem>
+      <gmd:referenceSystemIdentifier>
+        <gmd:RS_Identifier>
+          <gmd:code>
+            <gco:CharacterString>2193</gco:CharacterString>
+          </gmd:code>
+        </gmd:RS_Identifier>
+      </gmd:referenceSystemIdentifier>
+    </gmd:MD_ReferenceSystem>
+  </gmd:referenceSystemInfo>
+  <gmd:identificationInfo>
+    <gmd:MD_DataIdentification>
+      <gmd:citation>
+        <gmd:CI_Citation>
+          <gmd:title>
+            <gco:CharacterString>NZ Building Outlines (Full History)</gco:CharacterString>
+          </gmd:title>
+          <gmd:date>
+            <gmd:CI_Date>
+              <gmd:date>
+                <gco:Date>2019-04</gco:Date>
+              </gmd:date>
+              <gmd:dateType>
+                <gmd:CI_DateTypeCode codeList="http://asdd.ga.gov.au/asdd/profileinfo/gmxCodelists.xml#CI_DateTypeCode" codeListValue="creation">creation</gmd:CI_DateTypeCode>
+              </gmd:dateType>
+            </gmd:CI_Date>
+          </gmd:date>
+        </gmd:CI_Citation>
+      </gmd:citation>
+      <gmd:abstract>
+        <gco:CharacterString>
+
+This dataset consists of relationship table about polygon outlines representing buildings within mainland New Zealand. A building outline is a 2D representation of the roof outline of a building which has been classified from LINZ aerial imagery using a combination of automated and manual processes to extract and orthogonalise a building roof outline. This dataset does not include building outline geometry, but instead provides the linkages in building outline id's from different aerial imagery sets in the [NZ Building Outlines Full History] (https://data.linz.govt.nz/layer/xxxxx-nz-building-outlines-full-history) dataset when representations of building outlines change from single outlines split into multiple outlines, and from multiple outlines merged as single outlines.
```

Comment:
Suggested rewording of the abstract:

This dataset provides the relationships between building outlines within mainland New Zealand when buildings are split or merged.

A building outline is a 2D representation of the roof outline of a building which has been classified from LINZ aerial imagery using a combination of automated and manual processes to extract and refine a building roof outline.

**Data vintage**
This dataset does not include building outline geometry, but provides the linkages in building outline id's from different aerial imagery sources in the [NZ Building Outlines Full History] (https://data.linz.govt.nz/layer/xxxxx) dataset when building outlines change from single outlines split into multiple outlines, and from multiple outlines merged as single outlines.

This dataset will be updated with changes in building id relationships as new aerial imagery becomes available.
Please refer to the [NZ Building Outlines Data Dictionary] (https://nz-buildings.readthedocs.io/en/latest/introduction.html) for detailed metadata and information about this dataset.

**More building outlines data**
[NZ Building Outlines Full History] (https://data.linz.govt.nz/layer/xxxxx) - directly related to this dataset, it contains all combinations of building outlines from multiple years of imagery that have existed since the beginning of this dataset.

[NZ Building Outlines] (https://data.linz.govt.nz/layer/xxxxx) - provides current building outlines only derived from the latest LINZ aerial imagery.